e2e/test_access.py

```diff
@@ -1,13 +0,0 @@
-from __future__ import absolute_import
-from unittest import TestCase
-
-import requests
-
-class AccessTestCase(TestCase):
-
-    def setUp(self):
-        self.jenkins_url = 'http://localhost:8080'
-
-    def test_cli_access(self):
-        cli_status_code = requests.get("{}/cli".format(self.jenkins_url)).status_code
-        assert cli_status_code == 404
```

Comment:
Just to be clear: in the newer version of Jenkins that we are now using, the CLI (api that responds to requests from the local jenkins-cli jar file) is no longer available, correct? That's why we don't need to test for it?

Reply:
Correct. It's been removed.